modules/end-user-guide/nav.adoc

```diff
@@ -1,49 +1,49 @@
 ** xref:mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container.adoc[]
 ** xref:authenticating-on-scm-server-with-a-personal-access-token.adoc[]
-* xref:authoring-devfiles.adoc[]
-** xref:authoring-devfiles-version-1.adoc[]
-** xref:authoring-devfiles-version-2.adoc[]
+// * xref:authoring-devfiles.adoc[]
+// ** xref:authoring-devfiles-version-1.adoc[]
+* xref:authoring-devfiles-version-2.adoc[]
 * xref:customizing-developer-environments.adoc[]
-** xref:what-is-a-che-theia-plug-in.adoc[]
+// ** xref:what-is-a-che-theia-plug-in.adoc[]
 ** xref:adding-a-vs-code-extension-to-a-workspace.adoc[]
-** xref:adding-a-vs-code-extension-to-the-che-plugin-registry.adoc[]
-** xref:publishing-metadata-for-a-vs-code-extension.adoc[]
-** xref:testing-a-visual-studio-code-extension-in-che.adoc[]
-** xref:using-alternative-ides-in-che.adoc[]
-*** xref:configuring-a-workspace-to-use-an-ide-based-on-the-intellij-platform.adoc[]
-**** xref:building-images-for-ides-based-on-the-intellij-platform.adoc[]
-**** xref:provisioning-the-jetbrains-offline-activation-code.adoc[]
-*** xref:support-for-theia-based-ides.adoc[]
-** xref:adding-tools-to-che-after-creating-a-workspace.adoc[]
+// ** xref:adding-a-vs-code-extension-to-the-che-plugin-registry.adoc[]
+// ** xref:publishing-metadata-for-a-vs-code-extension.adoc[]
+// ** xref:testing-a-visual-studio-code-extension-in-che.adoc[]
+// ** xref:using-alternative-ides-in-che.adoc[] // TO REWRITE
+// *** xref:configuring-a-workspace-to-use-an-ide-based-on-the-intellij-platform.adoc[] // TO REWRITE
+// **** xref:building-images-for-ides-based-on-the-intellij-platform.adoc[] // TO REWRITE
+// **** xref:provisioning-the-jetbrains-offline-activation-code.adoc[] // TO REWRITE
+// *** xref:support-for-theia-based-ides.adoc[] // TO REWRITE
+// ** xref:adding-tools-to-che-after-creating-a-workspace.adoc[] // REMOVE
 ** xref:using-private-container-registries.adoc[]
-* xref:using-artifact-repositories-in-a-restricted-environment.adoc[]
-** xref:using-maven-artifact-repositories.adoc[]
-** xref:using-gradle-artifact-repositories.adoc[]
-** xref:using-python-artifact-repositories.adoc[]
-** xref:using-go-artifact-repositories.adoc[]
-** xref:using-nuget-artifact-repositories.adoc[]
-** xref:using-npm-artifact-repositories.adoc[]
+// * xref:using-artifact-repositories-in-a-restricted-environment.adoc[] // TO REWRITE
+// ** xref:using-maven-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-gradle-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-python-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-go-artifact-repositories.adoc[] // TO REWRITE 
+// ** xref:using-nuget-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-npm-artifact-repositories.adoc[] // TO REWRITE
 * xref:troubleshooting-che.adoc[]
 ** xref:viewing-che-workspaces-logs.adoc[]
 ** xref:investigating-failures-at-a-workspace-start-using-the-verbose-mode.adoc[]
```

Comment:
does this do anything other than opening the console log tab? when loading doesn't work, it's not very verbose... troubleshooting usually then involves going to the admin console, opening the pod, and reading ITS logs to see what went wrong. the dashboard doesn't expose enough of the pod log to be useful. @l0rd wondering if this section ought to be replaced with content about how to find your workspace's pod in the admin console and then debug via that... ?